poc/understanding/README.md

```diff
@@ -1 +1,31 @@
 ## Understanding
+
+##F23.String Similarity Algorithms
```

Comment:
Markdown formatting is not fully correct: https://github.com/artiso-solutions/CoVoX/tree/stringsimilarity/poc/understanding